server.js

```diff
@@ -22,12 +33,17 @@ server.use(cors({
     credentials: true}));
 
 
-server.use(helmet());
-server.use(cors());
-server.use(express.json());
-server.use(morgan(':method :url :status :res[content-type] - :response-time ms'))
 
+server.use('/users/', usersRouter)
 server.use('/auth', authRouter);
 server.use("/" , welcomeRouter)
-
+/*
+server.use((err, req, res, next) => {
+    console.log("Error:", err);
+  
+    res.status(500).json({
+      message: "Well, that's broken..."
+    });
+  });
+*/
```

Comment:
Do not commit commented out code.
This is known as [zombie code](https://www.codelord.net/2011/10/28/fight-zombie-code).
If you are committing frequently then any code you would comment out should already be in your git history for you to access later should you need it thus making commenting it out redundant.

I will not merge this PR until this code has either been uncommented or removed.